src/sc2api/sc2_client.cc

```diff
@@ -86,7 +86,7 @@ class ObservationImp : public ObservationInterface {
 
     // Game state info.
     UnitPool unit_pool_;
-    std::unordered_map<Tag, Unit> units_previous_map_;
+    std::unordered_map<Tag, Unit> units_previous_map_; // this is now unused 
```

Comment:
If it is unused, can we remove it?

Reply:
Unless you see a reason, it can be deleted. There is another reference to it that will also need to be deleted. I can do that after this PR is resolved

Reply:
Ah, ok. I'll check it.

Reply:
I deleted this in the branch on my fork. I (attempted) to rebase to the same commit used in the PR. Let me know if you cannot see it and, if not, what to do to fix it